doc/oemof_solph.rst

```diff
@@ -432,11 +432,60 @@ Furthermore, an efficiency for loading, unloading and a capacity loss per time i
         capacity_loss=0.001, nominal_capacity=50,
         inflow_conversion_factor=0.98, outflow_conversion_factor=0.8)
 
+
+Using an investment object with the GenericStorage component
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+Based on the `GenericStorage` object the `GenericInvestmentStorageBlock` adds two main investment possibilities.
+
+    *	Invest into the flow parameters e.g. a turbine or a pump
+    *	Invest into capacity of the storage  e.g. a basin or a battery cell
+    
+Investment in this context refers to the value of the variable for the 'nominal_value' (installed capacity) in the investment mode. 
+    
+As an addition to other flow-investments, the storage class implements the possibility to couple or decouple the flows 
+with the capacity of the storage. 
+Three parameters are responsible for connecting the flows and the capacity of the storage:
+
+    *	' `invest_relation_input_capacity` ' fixes the input flow investment to the capacity investment. A ratio of ‘1’ means that the storage can be emptied within one time-period. 
```

Comment:
shouldn't it be: 'filled within one timestep'?

Reply:
Is timestep already possible in this context or shall we use time step?